auth/loginHandler.js

```diff
@@ -0,0 +1,19 @@
+const bcrypt = require("bcrypt");
+
+const { getUserByEmail, addUserToken } = require("../controllers/users");
+const issueToken = require("./issueToken");
+
+const loginHandler = async (email, incomingPassword) => {
+  const user = await getUserByEmail(email);
+  const userPassword = user.password;
+  const result = bcrypt.compareSync(incomingPassword, userPassword);
+  if (result) {
+    const token = issueToken(user);
+    await addUserToken(user._id, token);
+    return token;
+  } else {
+    throw "Invalid credentials";
```

Comment:
To jest błąd, sugerowałbym zwracać nowy error zamiast stringa

`throw new Error("Invalid credentials)`

Reply:
Ok zmienione na: "

```suggestion
  throw new Error("Invalid credentials");
```